rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,153 @@
+### DCP PR:
+***Leave this blank until the RFC is approved then the Author(s) must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc\#\](https://github.com/HumanCellAtlas/dcp-community/pull/\<PR\#\>)`
+
+# Technical Support SOP
+
+## Summary
+
+This process RFC describes how the DCP handles technical support requests through the use of the Zendesk application and other processes.
+
+## Author(s)
+
+`[Kevin Osborn](mailto:kosborn2@ucsc.edu)`
+
+`[Jodi Hirschman](mailto:jhirsch@broadinstitute.org)`
+
+`[Matthew Spier](mailto:mspeir@ucsc.edu)`
+
+`[Mallory Freeberg](mailto:mfreeberg@ebi.ac.uk)`
+
+## Shepherd
+
+***Leave this blank. This role is assigned by DCP PM to guide the Author(s) through the RFC process.*
+
+## Motivation
+
+Beta 2 testing will begin soon and we’ve decided to collect feedback
+through our Zendesk ticketing system. We therefore need to quickly
+determine a process for handling larger numbers of Zendesk tickets than
+we have received to date. Several of us have been thinking about this
+for awhile and drew up an initial SOP that we refined at the last DCP
+F2F (Feb. 2019); we present it here as this RFC. 
+
+### **User Stories**
+
+1.  > As a researcher new to the DCP, I would like to get a timely
+    > answer to my question so that I can have a successful experience
+    > submitting, exploring, downloading, or analyzing HCA data.
+
+2.  > As a DCP PM, PO or PL, I would like to be assured that Zendesk
+    > tickets are being responded to in a timely manner so that I can
+    > focus on my own tasks without having to micromanage responses.
+
+3.  > As a DCP team member, I would like to know whether or not I should
+    > be responding to a Zendesk ticket so that I can either respond in
+    > a timely manner or be assured that someone else is responding in a
+    > timely manner.
+
+## Scientific "guardrails" \[optional\]
+
+
+## **Detailed Design**
+
+This section describes the flow of an issue through the support state
+diagram presented below.
+
+![](../images/0000-Zendesk-triage-SOP-diagram.jpeg)
```

Comment:
This is great, would it be possible to add to the secondary support step some color on "Export to Github"? Sometimes it's not 1:1, sometimes it links to an existing feature request with a subsequent record, sometimes it becomes a bug, or an issue request, or something else. Happy to help give an example as well of how we can prioritize (P1/P2/P3).

Reply:
I think an image here is a great idea. I have a few questions about the actual flow, though.

1. "User Issue Status: New" should technically not have any arrows coming into it, as it is the single input to the diagram. An issue is only "new" once; after someone responds to it at all, it is now "open". 
1. There are some answers missing from some of the arrows coming out of the diamond entities. 
1. There is a hanging arrow "Give solution to user" at the bottom of the diagram that needs to point to "closed" or something equivalent.
1. For some rectangle entities (not sure what these entities represent), there are multiple arrows coming out but it's unclear which arrow should be taken.

I'm happy to work on editing the diagram if someone could share an editable file with me :)

Reply:
@malloryfreeberg Are you able to edit this doc?: https://www.lucidchart.com/documents/edit/f7cc3f5e-c306-40bc-8063-6aa3fb75e179/0?shared=true&
It's an updated version from Kevin- updated based on your list.  Feel free to edit more!

Reply:
I can view the doc. Requested access to edit.

Changes look good!